spec/index.md

```diff
@@ -1659,15 +1675,15 @@ For each canister, the IC keeps track of "certified data", a canister-defined bl
 
     The canister can update the certified data with this call. The passed data must be no larger than 32 bytes. This can be used any number of times.
 
-When executing a query method via a query call (i.e. in non-replicated state), the canister can fetch a certificate that authenticates to third parties the value last set via `ic0.certified_data_set`.
+When executing a query or composite query method via a query call (i.e. in non-replicated mode), the canister can fetch a certificate that authenticates to third parties the value last set via `ic0.certified_data_set`. The certificate is not available in composite query method callbacks and in query and composite query methods evaluated on canisters other than the target canister of the query call.
```

Comment:
"other than the target canister of the query call"
I'm curious, why the restriction?

Reply:
It's a deficiency of the current implementation afaik